libass/ass/meson.build

```diff
@@ -1,4 +0,0 @@
-# Hack to work around https://github.com/mesonbuild/meson/issues/2546
-foreach header : libass_headers
-    configure_file(copy: true, input: header, output: '@PLAINNAME@')
-endforeach
```

Comment:
It's not a meson bug.

Canonically, to depend on libass, one uses the following includes:

```c
#include <ass/ass.h>
#include <ass/ass_types.h>
```

However, those paths don't exist in the repository (instead, libass/ass.h etc do). It is therefore impossible to build against an uninstalled version of libass e.g. a local static build, or libass as a subproject, unless you either:

- have libass copy the headers over into a compatible uninstalled structure as part of the build
- use a define to tell your code whether your libass dependency is uninstalled, and select between including ass/ass.h vs including libass/ass.h

Aegisub cannot "work around" the fact that the compiler requires an on-disk file "ass.h" inside a directory called "ass".

Some projects store all their public headers inside an include/ directory instead of mingled in with the library code, and never have to deal with this. Some projects do the copy. Some projects don't use a directory structure for their headers.